scripts/migrate_email_templates.js

```diff
@@ -0,0 +1,68 @@
+"use strict";
+const Services = {
+    env: require("../services/env.service"),
+    db: require("../services/database.service")
+}
```

Comment:
Do we usually do requires inside an object? If not, I think it's a bit more intuitive to just say
```
const env = require()
const db = require()
```